tests/TestCase/ORM/CompositeKeysTest.php

```diff
@@ -673,6 +673,73 @@ public function testFindThreadedCompositeKeys()
     }
 
     /**
+     * Tets that loadInto() is capable of handling composite primary keys
+     *
+     * @return void
+     */
+    public function testLoadInto()
+    {
+        $table = TableRegistry::get('SiteAuthors');
+        $tags = TableRegistry::get('SiteTags');
+        $table->hasMany('SiteArticles', [
+            'foreignKey' => ['author_id', 'site_id'],
+        ]);
+
+        $author = $table->get([1, 1]);
+        $result = $table->loadInto($author, ['SiteArticles']);
+        $this->assertSame($author, $result);
```

Comment:
That is done below, when I compare to a normal contain call. But I can add the assert for clarity